src/data-browser/graphs/highchartsConfig.js

```diff
@@ -151,25 +152,39 @@ export const deriveHighchartsConfig = ({
 
   config.yAxis = yAxis.map((yTitle, _yIdx) => ({
     title: { text: yTitle },
-    labels: hmda_charts.styles.axisLabel,
+    labels: {
+      ...hmda_charts.styles.axisLabel,
+      // Formatter used to add "K" and "M"
+      formatter: (tick) => {
+        if (!tick) return
+
+        if (tick.value > 999 && tick.value < 1000000) {
+          return (tick.value / 1000).toFixed(0) + "K" // convert to K for number from > 1000 < 1 million
+        } else if (tick.value >= 1000000) {
+          return (tick.value / 1000000).toFixed(1) + "M" // convert to M for number from > 1 million
+        } else if (tick.value < 900) {
+          return tick.value // if value < 1000, nothing to do
+        }
```

Comment:
What happens if the value is `950`?
 
I don't think we need any comparison here.  Just, in all other cases, return the plain value. 
```suggestion
        } else {
          return tick.value // if value < 1000, nothing to do
        }
```

Reply:
Code has been updated!